src/main/autoUpdate.js

```diff
@@ -16,5 +17,14 @@ export default (notify: SendFunction) => {
 }
 
 export function quitAndInstall() {
-  autoUpdater.quitAndInstall()
+  setImmediate(() => {
+    const browserWindows = BrowserWindow.getAllWindows()
+
+    app.removeAllListeners('window-all-closed')
+    browserWindows.forEach(browserWindow => {
+      browserWindow.removeAllListeners('close')
+    })
```

Comment:
curious about what was the problem here?

Reply:
(add a comment in sourcecode)